components/post_view/post_header/index.ts

```diff
@@ -24,14 +31,26 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
     }
 
     const user = getUser(state, ownProps.post.user_id);
+    const currentUser = getCurrentUser(state);
+    const customStatus = user ? getCustomStatus(state, user.id) : {};
     const isBot = Boolean(user && user.is_bot);
 
     return {
         enablePostUsernameOverride,
         isBot,
         overwriteIcon,
         isGuest: Boolean(user && isGuest(user)),
+        customStatus,
+        userID: currentUser.id,
```

Comment:
Rename to `currentUserID`

```suggestion
        currentUserID: currentUser.id,
```